Dockerfile.test

```diff
@@ -31,10 +31,10 @@ RUN /kairos-init validate
 FROM ${BASE_IMAGE} AS default
 ARG MODEL=generic
 ARG TRUSTED_BOOT=false
-ARG KUBERNETES_PROVIDER=k3s
-ARG KUBERNETES_VERSION=latest
+ARG PROVIDER_NAME
+ARG PROVIDER_VERSION=latest
 ARG VERSION=v0.0.1
 ARG FIPS=false
 
 COPY --from=kairos-init /kairos-init /kairos-init
-RUN /kairos-init -l debug -m "${MODEL}" -t "${TRUSTED_BOOT}" -k "${KUBERNETES_PROVIDER}" --k8sversion "${KUBERNETES_VERSION}" --version "${VERSION}" "$(if [ "${FIPS}" = "true" ]; then echo "--fips"; fi)" && /kairos-init validate -t "${TRUSTED_BOOT}" && rm /kairos-init
\ No newline at end of file
+RUN /kairos-init -l debug -m "${MODEL}" -t "${TRUSTED_BOOT}" -k "${PROVIDER_NAME}" --k8sversion "${PROVIDER_VERSION}" --version "${VERSION}" "$(if [ "${FIPS}" = "true" ]; then echo "--fips"; fi)" && /kairos-init validate -t "${TRUSTED_BOOT}" && rm /kairos-init
```

Comment:
it's a bit weird that the flag is -k and -k8sversion and the content is generic, but i guess that's what your final comment is about. How about, `--provider` and `--provider-version` and you pass it multiple times, so if you wanted two you'd have to do `--provider=kairos` and `--provider-version=v1.2.3` and `--provider=other`?

Reply:
yes, that is what i was thinking about but  then the problem is to construct  map of the provider stuff. Like if you pass provider x and provider y and provider-version z then making it match with the second provider? Its a mess, ugh

Reply:
I'd say is the responsibility of the user to pass them accordingly, if they pass one version they must pass all oversions and so. IMO this is better than the string split by commas